tests/admin_views/admin.py

```diff
@@ -574,8 +574,7 @@ class ComplexSortedPersonAdmin(admin.ModelAdmin):
     ordering = ('name',)
 
     def colored_name(self, obj):
-        return '<span style="color: #%s;">%s</span>' % ('ff00ff', obj.name)
-    colored_name.allow_tags = True
+        return format_html('<span style="color: #{};">{}</span>', 'ff00ff', obj.name)
```

Comment:
existing issue, but interpolating `'ff00ff'` seems a bit odd, doesn't it? I'd just include that in the original string.